### PR TITLE
Response style changed

### PR DESCRIPTION
**Pull request by gautamramuvel**

### CODE REVIEW

**Comment by gauthamzz** [comment]
files are again changed in so many unwanted places. This issue requires changing of one or two lines at max :smile:

File: background.js [hunk 1]
```diff
@@ -179,89 +179,97 @@ $(document).ready(function () {
...
-
-
+    
+    //sending the data to server
+    function send() {
+        //alert('you said ' + txt);
+        txt = txt.replace('hey ', '');
+        //setResponse(txt);
+        //console.log('you said ' + txt);
+        alert(txt);
```
Comment (gauthamzz) [inline background.js:189]: why are you alrerting ?

File: background.js [hunk 2]
```diff
@@ -179,89 +179,97 @@ $(document).ready(function () {
...
-        "Authorization": "Bearer " + accessToken
-      },
-      data: JSON.stringify({ query: txt, lang: "en", sessionId: "somerandomthing" }),
-      success: function (data) {
-        // alert("intent " + data.result.metadata.intentName);
-                  if (data.result.metadata.intentName === "youtube") {
-                   searchYoutube(data.result.parameters.any);
-          // chrome.tabs.create({ 'url': 'https://www.youtube.com/results?search_query=' + data.result.parameters.any });
+    //sending the data to server
+    function tasks() {
```
Comment (gauthamzz) [inline background.js:194]: why is this even changed :small_airplane:

**Review by gauthamzz — CHANGES_REQUESTED** [review]

**Comment by gautamramuvel** [comment]
It's the same code

**Comment by gauthamzz** [comment]
Are you working on this ?

**Comment by gautamramuvel** [comment]
It's done..

**Comment by gautamramuvel** [comment]
I'm yet to push

**Pull request closed without merging**